build-packages/eslint-plugin-scandipwa-guidelines/lib/rules/use-namespace.js

```diff
@@ -163,18 +164,17 @@ const preparePackageName = (packageName) => {
 };
 
 const generateNamespace = (node, context) => {
-    const filename = context.getFilename();
-    const splitted = filename.split('src');
-    const toFile = splitted.pop();
-    const toPackage = path.normalize(splitted.join('src'));
-    const { name: packageName } = getPackageJson(toPackage);
+    const filePath = context.getFilename();
+    const modulePath = walkDirectoryUp(filePath).pathname;
```

Comment:
Interesting mix of destructure on line 170 and here. Why so?

Reply:
Please elaborate.